### ISSUE TITLE
Support inlay hints.

### ISSUE THREAD

**Issue by byehack** [microsoft/pylance-release#1559]
inlay hints can be usefull for:
show parameters name of callable objects and functions.
better showing errors/warnings.
showing type hints.

**Comment by jerryc05** [comment]
I totally agree. Inlay hints from the JS/TS plugins are awesome and Python should also deserve this cool feature with the type inferred by Pylance.

**Comment by judej** [comment]
Moving this issue to discussion as an enhancement request for comments and upvotes.
